lib/l10nUtil.js

```diff
@@ -87,6 +87,7 @@ module.exports.rebaseBraveStringFilesOnChromiumL10nFiles = (path) =>
         .replace('<include name="IDR_MD_HISTORY_SIDE_BAR_HTML"', '<include name="IDR_MD_HISTORY_SIDE_BAR_HTML" flattenhtml="true"')
         .replace(pageVisibility, bravePageVisibility + pageVisibility)
         .replace(/settings_chromium_strings.grdp/g, 'settings_brave_strings.grdp')
+        .replace(/Automatically sends usage statistics and crash reports to Brave/g, 'Automatically sends crash reports to Brave')
```

Comment:
I merged already so need another PR, but I think this needs to be at the bottom.